File: 1). DSA/1). Data Structures/01). Array/C++/_011)_Largest_Number.cpp
#include<iostream>
using namespace std;

int getLargest(int arr[], int n)
{
    for(int i=0; i<n; i++)
    {
        bool flag = true;
        for(int j=1; j<n; j++)
        {
            if(arr[j] > arr[i])
            {
                flag = false;
                break;
            }
        }
        if(flag==true)
            return i;
    }
    return -1;
}

int main()
{
    int arr[] = {10, 5, 20, 8}, n=4;
    cout<<"Index = "<<getLargest(arr, n)<<endl;
}

File: 1]. DSA/1]. Data Structures/01]. Array/JavaScript/_011)_Largest_Number.js
function getLargest(arr) {
    for (let i = 0; i < arr.length; i++) {
        let flag = true;
        for (let j = 1; j < arr.length; j++) {
            if (arr[j] > arr[i]) {
                flag = false;
                break;
            }
        }
        if (flag == true) return i;
    }
    return -1;
}

let arr = [10, 5, 20, 8];
let index = getLargest(arr);
console.log(`Index = ${index}
Largest Number in [${arr}] is ${arr[index]}`);
